pkg/networkservice/core/trace/common.go

```diff
@@ -17,40 +17,41 @@
 package trace
 
 import (
+	"context"
 	"reflect"
 	"strconv"
 
+	"github.com/networkservicemesh/sdk/pkg/tools/logger"
+
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/reflect/protoreflect"
-
-	"github.com/networkservicemesh/sdk/pkg/tools/spanhelper"
 )
 
-func logRequest(span spanhelper.SpanHelper, request proto.Message) {
-	connInfo, ok := trace(span.Context())
+func logRequest(ctx context.Context, request proto.Message) {
+	connInfo, ok := trace(ctx)
 	if ok && !proto.Equal(connInfo.Request, request) {
 		if connInfo.Request != nil && connInfo.Request.ProtoReflect().Descriptor().FullName() == request.ProtoReflect().Descriptor().FullName() {
 			requestDiff, hadChanges := Diff(connInfo.Request.ProtoReflect(), request.ProtoReflect())
 			if hadChanges {
-				span.LogObject("request-diff", requestDiff)
+				logger.Log(ctx).Infof("request-diff=%v", requestDiff)
```

Comment:
We loose semantic here, opentracing and jaeger support working with object structure if object is passed as is to it, but here we convert it to string representation and will loose this fancy functionality on jaeger UI.